### PR TITLE
Updated foreman_discovery to 5.0.1 (DEB)

### PR DESCRIPTION
**Pull request by lzap**

### CODE REVIEW

File: plugins/ruby-foreman-discovery/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,15 @@
+ruby-foreman-discovery (5.0.1-1) stable; urgency=low
```
Comment (domcleal) [inline plugins/ruby-foreman-discovery/debian/changelog]: Duplicate lines.

**Comment by domcleal** [comment]
Foreman seems to be consistently failing to install, though I don't see why right now.

<pre>
Setting up foreman (9999-wheezy+scratchbuild+201602251134) ...
dpkg: error processing foreman (--configure):
 subprocess installed post-installation script returned error exit status 6
</pre>

(http://ci.theforeman.org/job/packaging_build_deb_plugin/983/)

I'll try and reproduce this independently later.

**Comment by lzap** [comment]
Removed the extra entry, isn't this a dep issue?

```
dpkg: error processing foreman (--configure):
 subprocess installed post-installation script returned error exit status 6
dpkg: dependency problems prevent configuration of foreman-sqlite3:
 foreman-sqlite3 depends on foreman; however:
  Package foreman is not configured yet.

dpkg: error processing foreman-sqlite3 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of ruby-foreman-deface:
 ruby-foreman-deface depends on foreman; however:
  Package foreman is not configured yet.

dpkg: error processing ruby-foreman-deface (--configure):
```

I havent touched deps in the release, deface was always there.

**Comment by mmoll** [comment]
[test] again

**Comment by mmoll** [comment]
strange, I can't reproduce this locally...

**Comment by domcleal** [comment]
@lzap the message about dependencies is just saying that it can't install a later package as configuration of the first one (its dependency) failed.  I think the underlying issue is the configuration of the main `foreman` package that's failing.

**Comment by lzap** [comment]
Thanks for info, I will hold with annoucing the release until this is resolved.

**Comment by domcleal** [comment]
The problem seems to be in ruby-foreman-deface on 1.11+.  I added some debugging and found:

<pre>
+ /usr/bin/foreman-ruby /usr/bin/bundle install --path ./vendor/ --local --no-prune
Resolving dependencies...
Some gems seem to be missing from your vendor/cache directory.
Could not find gem 'polyglot (>= 0) ruby', which is required by gem 'deface (=
1.0.2) ruby', in any of the sources.
</pre>

I can reproduce this on a regular installation by installing deface, but not in pbuilder for some reason.

polyglot was removed as a 'mail' dependency recently, and in develop with Rails 4 we're getting a newer version of mail versus Rails 3.  I think we need to add polyglot into the deface package (I'll send a PR).

**Comment by mmoll** [comment]
[test] again, after #1044 has been merged and built

**Comment by lzap** [comment]
Looks green!

**Comment by domcleal** [comment]
Merged as c7a77ca, 3607e1c, thanks @lzap.

Building at http://ci.theforeman.org/job/packaging_build_deb_plugin/1000/ and http://ci.theforeman.org/job/packaging_build_deb_plugin/1001/.

**Pull request closed without merging**